### ISSUE TITLE
Increase flashlight brightness to maximum by default instead of 50%.

### ISSUE THREAD

**Issue by ghost** [GrapheneOS/os_issue_tracker#4473]
GrapheneOS should change this AOSP behaviour. Currently the only way to have the flashlight shine with max brightness is to open it from the Camera app.

**Comment by WaleedMortaja** [comment]
I did not know you can change the flashlight brightness. If it is possible, I would suggest it to be configurable from the quick tile (long/hold click the flash light, for example, to enter a config page with slider to change the brightness level)

**Comment by WaleedMortaja** [comment]
I do not think it would be hard to make brightness adjustable from a UI view point. Since it can be configured by changing some variable as you mention. It should be manageable to add a slider that changes that variable, the same way as we have sliders for changing sound levels or brightness or others. So why to force the user on some brightness level then?
